test-smoke/electron/test/renderer.ts

```diff
@@ -60,11 +60,19 @@ console.log(webFrame.getZoomLevel());
 webFrame.setVisualZoomLevelLimits(50, 200);
 webFrame.setLayoutZoomLevelLimits(50, 200);
 
-webFrame.setSpellCheckProvider('en-US', true, {
-	spellCheck: text => {
-		return !(require('spellchecker').isMisspelled(text));
-	}
-});
+webFrame.setSpellCheckProvider('en-US', {
+  spellCheck (words, callback) {
+    setTimeout(() => {
+      let misspeltWords = []
+      for (let word of words) {
+        if (require('spellchecker').isMisspelled(word)) {
```

Comment:
* `require()` has a nontrivial cost, so it's better to move it outside of this loop into a local `const`
* `Array.prototype.filter()` would be useful here